### PR TITLE
Split and cleanup terminator test policies

### PR DESCRIPTION
**Pull request by jillr**
Remove unused test policies for polly, iot, and rekognition. These had been
added in support of new module PRs which were never merged.

Rename big-data.yaml to data-services.yaml and migrate database policies
from compute.yaml and application-services.yaml to reduce the size of these
policy files (both approaching 6144 character max).

Add a test to deploy-test-policy.yml for character count of the policy json
to make it easier for contributors to know if a policy file is over sized.

Unmerged new module PRs:
Polly: https://github.com/ansible/ansible/pull/40889
Rekognition: https://github.com/ansible/ansible/pull/40768
IOT: https://github.com/ansible/ansible/pull/32826

application-services.yaml will exceed 6144 characters if we were to merge https://github.com/mattclay/aws-terminator/pull/93, this policy refactor will unblock that change.

@mattclay When renaming policies in the past have we preferred to manually delete the old policy, like for ansible-core-ci-test-big-data-dev and ansible-core-ci-test-machine-learning-dev?  I'm not seeing any playbooks here oriented to cleanup.

### CODE REVIEW

**Comment by mattclay** [comment]
@jillr I'm OK with manual cleanup of the old policies, since it isn't a frequent occurrence.

**Comment by jillr** [comment]
Test deployed new policies to stage=dev and checked cleanup.py, should be good now.
Have not yet deleted old policies, will do so when this is merged and deployed to both prod and stage (I've redeployed master to stage=dev in the interest of having things the way they will be expected to be).

ready_for_review

File: aws/deploy-test-policy.yml [hunk 1]
```diff
@@ -33,6 +33,12 @@
       tags: iam
       set_fact:
         policies: '{{ lookup("fileglob", "policy/*.yaml", wantlist=True) | map("basename") | map("regex_findall", "^(.*?)\.yaml") | list | flatten }}'
+    - name: Check policies do not exceed AWS size limits
```
Comment (tremble) [inline aws/deploy-test-policy.yml:36]: Not a blocker but we should probably push this into a pre-merge check.

**Comment by jillr** [comment]
New policies deployed to prod,dev. New terminator deployed to prod.  Deleted policies via console:
```
ansible-core-ci-test-machine-learning-dev
ansible-core-ci-test-machine-learning-prod
ansible-core-ci-test-big-data-dev
ansible-core-ci-test-big-data-prod
```

**Pull request merged**